res/values/themes.xml

```diff
@@ -137,4 +137,10 @@
     <item name="emoji_button_bg">#ffffff</item>
     <item name="emoji_color">#000000</item>
   </style>
+  
+<style name="Darkv2" parent="@style/BaseTheme"><item name="android:isLightTheme">false</item><item name="colorKeyboard">#ff121212</item><item name="colorKey">#ff434343</item><item name="colorKeyActivated">#ff1a237e</item><item name="colorLabel">#ffffffff</item><item name="colorLabelActivated">#ff000000</item><item name="colorLabelLocked">#ffffffff</item><item name="secondaryLightOffset">-0.2</item><item name="colorSubLabel">#ffaaaaaa</item><item name="keyBorderWidth">0.0dip</item><item name="keyBorderWidthActivated">0.0dip</item><item name="keyBorderColorBottom">#ff121212</item><item name="emoji_button_bg">?colorKeyActivated</item><item name="emoji_color">#ffffffff</item></style>
```

Comment:
Could you explain the point of the darkv2 and lightv2 themes ? I'd prefer to improve the existing themes rather than adding slight variations.

Reply:
Well, I do it separately so that you can visualize them separately, and if you like them, they are changed for the old ones, or leave them aside or eliminate them :v

> ¿Podría explicar el punto de los temas darkv2 y lightv2? Preferiría mejorar los temas existentes en lugar de agregar ligeras variaciones.

Well, I do it separately so that you can visualize them separately, and if you like them, they are changed for the old ones, or leave them aside or eliminate them :v